tests-e2e/core/managed_environment_test.go

```diff
@@ -994,7 +994,17 @@ var _ = Describe("Environment E2E tests", func() {
 			Expect(managedEnvCR.Spec.APIURL).To(Equal(dt.Spec.KubernetesClusterCredentials.APIURL))
 			Expect(managedEnvCR.Spec.ClusterCredentialsSecret).To(Equal(dt.Spec.KubernetesClusterCredentials.ClusterCredentialsSecret))
 			Expect(managedEnvCR.Spec.AllowInsecureSkipTLSVerify).To(Equal(dt.Spec.KubernetesClusterCredentials.AllowInsecureSkipTLSVerify))
-		})
+
+			// In both cases the ManagedEnv Namespaces value should match the value from DT
```

Comment:
I updated the test to test both the namespace empty, and namespace non empty case, via a `DescribeTable`